src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
```

Comment:
add class, don't use elements for styling

Reply:
ok